Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,62 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation Github Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the
+joint operational direction of the Node.js Technical Steering Committee (TSC)
+and Node.js Foundation Community Committee (CommComm).
+
+## Organization Roles
+
+### Owners
+
+The Node.js TSC Director, TSC Facilitator, and the two elected individual
+member representatives shall be granted Owner permissions within the Node.js
+GitHub Organization.
+
+Additional TSC, CommComm, and Core Technical Committee (CTC) members may be
+granted Owner permissions by request. Assuming there are no objections from any
+current voting TSC or CommComm member, such requests are approved automatically
+after 72 hours. If any objection is made, the request may be moved to a single
+vote in which a simple majority of both the TSC and CommComm is required to
+approve. Such requests must be posted as issues to either the nodejs/TSC or
+nodejs/community-committee repositories.
+
+Upon the completion of the TSC Director, TSC Facilitator, or elected individual
+member representative's terms, their Owner permissions within the GitHub
+organization shall be maintained *unless* the individual is also no longer a
+voting member of either the TSC or Community Committee.
+
+An individuals Owner permissions shall be automatically removed if they are no
+longer voting members of the TSC, CommComm, or CTC. Further, Owner permissions
+for any individual may be removed through single simple majority vote of both
+the TSC and CommComm.
+
+### Members
+
+GitHub users are added as members to the Node.js GitHub Organization when they
+are added to any Working Group. Organization Owners should add new members
+to the organization when requested by a Working Group.
+
+## Repositories
+
+Any organization member may request the creation of a new repository within the
+Node.js Foundation GitHub Organization. Provided there are no objections from
+any voting TSC or CommComm member, such requests are approved automatically
+after 72 hours. If any objection is made, the request may be removed to a single
+vote in which a simple majority of both the TSC and CommComm is required to
+approve. Such requests must be posted as issues to either the nodejs/TSC or
+nodejs/community-committee repositories.
```

Comment:
When someone makes a request, is the idea that the committee who received the request would become the owner of that repo? I may be overthinking this, but do we need to be concerned with someone requesting a new repo from one committee, getting turned down, and then requesting from the other committee as a way to get around the first?